ui/window.go

```diff
@@ -745,7 +749,13 @@ func NewWindow(app *tview.Application, session *discordgo.Session, readyEvent *d
 
 	window.userList = NewUserTree(window.session.State)
 
-	if config.Current.OnTypeInListBehaviour == config.SearchOnTypeInList {
+	// Disable search on type when in Vim mode. TODO add / shourtcut to search.
+	if window.app.VimMode.CurrentMode != vim.Disabled {
```

Comment:
Will toggling vim mode also toggle this?

Reply:
It will not. Thanks for the pointer